torchrl/data/utils.py

```diff
@@ -94,7 +94,7 @@ def expand_as_right(
         )
     for _ in range(dest.ndimension() - tensor.ndimension()):
         tensor = tensor.unsqueeze(-1)
-    return tensor.expand_as(dest)
+    return tensor.expand(dest.shape)
```

Comment:
curious as why change this? Maybe because `.expand_as` is not implemented for `_TensorDict` ?

Reply:
Exactly!